$NetBSD: patch-src_Makefile,v 1.2 2012/01/09 16:02:13 khorben Exp $

- rename the final binary to avoid conflicts with the default pre-processor

--- src/Makefile.orig	2011-06-16 15:14:36.000000000 +0000
+++ src/Makefile
@@ -64,13 +64,13 @@ install: $(TARGETS)
 	$(LN) -s -- libcpp.so.0.0 $(DESTDIR)$(LIBDIR)/libcpp.so.0
 	$(LN) -s -- libcpp.so.0.0 $(DESTDIR)$(LIBDIR)/libcpp.so
 	$(MKDIR) $(DESTDIR)$(BINDIR)
-	$(INSTALL) -m 0755 -- cpp $(DESTDIR)$(BINDIR)/cpp
+	$(INSTALL) -m 0755 -- cpp $(DESTDIR)$(BINDIR)/deforaos-cpp
 
 uninstall:
 	$(RM) -- $(DESTDIR)$(LIBDIR)/libcpp.a
 	$(RM) -- $(DESTDIR)$(LIBDIR)/libcpp.so.0.0
 	$(RM) -- $(DESTDIR)$(LIBDIR)/libcpp.so.0
 	$(RM) -- $(DESTDIR)$(LIBDIR)/libcpp.so
-	$(RM) -- $(DESTDIR)$(BINDIR)/cpp
+	$(RM) -- $(DESTDIR)$(BINDIR)/deforaos-cpp
 
 .PHONY: all clean distclean install uninstall
